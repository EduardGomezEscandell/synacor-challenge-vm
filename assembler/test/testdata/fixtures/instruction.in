set r0 0x15