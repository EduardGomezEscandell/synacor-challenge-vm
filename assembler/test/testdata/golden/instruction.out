<IDENTIFIER set> <REGISTER 0> <NUMBER 21> <EOL>
<END>
